src/parser.js

```diff
@@ -60,9 +60,10 @@ export class Parser {
   // Look for `token` in this.line[this.pos], and return success
   // if one is found. `token` is specified as a regular expression.
   matchToken(token, noSpace = false) {
-    const re = noSpace ? new RegExp('^((' + token + '))') : new RegExp('^((' + token + ')\\s*)');
+    const regexp = noSpace ? new RegExp('^((' + token + '))') :
```

Comment:
Yes, agreed. Reformatted.

On Sat, Aug 20, 2016 at 8:59 AM, Cyril Silverman <notifications@github.com>
wrote:

> In src/parser.js
> <https://github.com/0xfe/vexflow/pull/424#discussion_r75578219>:
>
> > @@ -60,9 +60,10 @@ export class Parser {
> >    // Look for `token` in this.line[this.pos], and return success
> >    // if one is found. `token` is specified as a regular expression.
> >    matchToken(token, noSpace = false) {
> > -    const re = noSpace ? new RegExp('^((' + token + '))') : new RegExp('^((' + token + ')\\s*)');
> > +    const regexp = noSpace ? new RegExp('^((' + token + '))') :
>
> Sorry more nitpicking :)
>
> I find that multi-line ternary are clearer when formatted like this:
>
> const regexp = noSpace
>   ? new RegExp('^((' + token + '))')
>   : new RegExp('^((' + token + ')\\s*)');
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/0xfe/vexflow/pull/424/files/e574a4fe89334b6b7b350ba68d7de417eb06607a..6c60b81b2df2db46b616194080d28f2bb1968ad7#r75578219>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOuk4vhafxEcvJwYAMlFdWy06zdH735ks5qhvoygaJpZM4Jo9qD>
> .
>



-- 
Mohit Muthanna [mohit (at) muthanna (uhuh) com]